sdk/python/feast/infra/online_stores/dynamodb.py

```diff
@@ -138,6 +138,45 @@ async def close(self):
     def async_supported(self) -> SupportedAsyncMethods:
         return SupportedAsyncMethods(read=True, write=True)
 
+    @staticmethod
+    def _table_tags(online_config, table_instance) -> list[dict[str, str]]:
+        online_tags = online_config.tags or {}
+        common_tags = [
+            {"Key": key, "Value": value} for key, value in online_tags.items()
+        ]
+        return common_tags + (
+            [
+                {"Key": key, "Value": value}
+                for key, value in table_instance.tags.items()
+                if key not in online_tags
+            ]
+            if table_instance.tags
+            else []
+        )
+
+    @staticmethod
+    def _update_tags(dynamodb_client, table_name: str, new_tags: list[dict[str, str]]):
+        table_arn = dynamodb_client.describe_table(TableName=table_name)["Table"][
+            "TableArn"
+        ]
+        current_tags = dynamodb_client.list_tags_of_resource(ResourceArn=table_arn).get(
+            "Tags"
+        )
+        if current_tags:
```

Comment:
if there are existing tags, remove those that will be changing w the update. either key no longer exists or the value changed